azure-iot-device/tests/common/pipeline/test_pipeline_stages_mqtt.py

```diff
@@ -258,30 +278,28 @@ def transport_function_throws_base_exception(params, stage, mocker, fake_base_ex
 
 @pytest.mark.parametrize("params", connection_ops + pubsub_ops)
 @pytest.mark.describe(
-    "MQTTTransportStage - .run_op() -- called with op that maps directly to protocol client library calls"
+    "MQTTTransportStage - .run_op() -- called with op that maps directly to transport calls"
```

Comment:
I also don't think there's any validation of the callbacks `on_published()`, `on_subscribed()`, `on_unsubscribed()` due to the messaging family of operations having to share complete testing infrastructure with the connection family of operations.